IronSoftware.Drawing/IronSoftware.Drawing.Common/CropRectangle.cs

```diff
@@ -28,6 +49,48 @@ public CropRectangle(int x, int y, int width, int height)
         /// The height of this Rectangle. The default is 0.
         /// </summary>
         public int Height { get; set; }
+        /// <summary>
+        /// Unit of measurement. The default is Pixels
+        /// </summary>
+        public MeasurementUnits Units
+        {
+            get;
+            set;
+        } = MeasurementUnits.Pixels;
+
+        /// <summary>
+        /// Convert this crop rectangle to the specified units of measurement using the specified DPI
+        /// </summary>
+        /// <param name="units">Unit of measurement</param>
+        /// <param name="dpi">DPI for conversion</param>
```

Comment:
Please use:
`        /// <param name="dpi">DPI (Dots per Inch) for conversion</param>`